### DISCUSSION TITLE
How can I fill up my rectangle with a color

### DISCUSSION THREAD

**Discussion by Dell-Optiplex-390** [CosmosOS/Cosmos#2090] [Q&A]
#### Area of Cosmos - What area of Cosmos are we dealing with?
I used canvas.DrawRectangle() and I want to fill up the rectangle with a color

#### Expected Behaviour - What do you think that should happen?
n/a

#### Actual Behaviour - What unexpectedly happens?
n/a

#### Reproduction - How did you get this error to appear?
n/a

#### Version - Were you using the User Kit or Dev Kit? And what User Kit version or Dev Kit commit (Cosmos, IL2CPU, X#)?
UserKit, C
![Screenshot 2022-02-05 130153](https://user-images.githubusercontent.com/80816207/152643230-78f86f79-e9ea-4bb4-9c3a-30d7bf19b223.png)
#

**Comment by valentinbreiz** [comment]
Hello, you can use the DrawFilledRectangle method

**Comment by Dell-Optiplex-390** [comment]
Thank you so much for helping, I have a small issue though, do you know what the co-ordinates are for putting the rectangle in the middle(x, y)?

**Reply by quajak** [reply]
You can get the width and height of the canvas and then simply divide by 2 to find the middle.
